docs/source/releasing.rst

```diff
@@ -44,7 +48,7 @@ Things to do for releasing:
 * push to PyPI::
 
     git clean -xdf   # maybe run 'git clean -xdn' first to see what it will delete
-    python3 setup.py sdist bdist_wheel
+    python3 -m build -ws
```

Comment:
Do not pass both `--sdist` and `--wheel`, because this causes wheels building from the Git checkout, as opposed to building wheels from sdist (as `pip wheel` does).
```suggestion
    python3 -m build
```